spacy/ml/models/coref_util.py

```diff
@@ -143,16 +143,18 @@ def create_head_span_idxs(ops, doclen: int):
 
 
 def get_clusters_from_doc(doc) -> List[List[Tuple[int, int]]]:
-    """Given a Doc, convert the cluster spans to simple int tuple lists."""
+    """Given a Doc, convert the cluster spans to simple int tuple lists. The
+    ints are char spans, to be tokenization independent.
+    """
     out = []
     for key, val in doc.spans.items():
         cluster = []
```

Comment:
```suggestion
        cluster = set()
```

Reply:
One of the failure modes of coref is to say `[John, John, John, John]` (exactly the same span) are all coreferences, so eliminating duplicate indices with a set would obscure that.